doc/nrf-bm/release_notes/release_notes_changelog.rst

```diff
@@ -119,6 +119,11 @@ Libraries
   * :ref:`lib_ble_service_nus` service.
   * Bluetooth LE Record Access Control Point library.
 
+* :ref:`lib_ble_adv` library:
+
+  * Updated the advertising and scan response data encoder provided in the :file:`include/bm/bluetooth/ble_adv_data.h`so that it can be used by enabling the :kconfig:option:`CONFIG_BLE_ADV_DATA` Kconfig option.
```

Comment:
```suggestion
  * Updated the advertising and scan response data encoder provided in the :file:`include/bm/bluetooth/ble_adv_data.h` file so that it can be used by enabling the :kconfig:option:`CONFIG_BLE_ADV_DATA` Kconfig option.
```